### ISSUE TITLE
cannot find com.google.android.maps.GeoPoint class after add this library.

### ISSUE THREAD

**Issue by SwiftyWang** [seiginonakama/BlockCanaryEx#7]
then I try to modify the configuration in the build.gradle and
added:
```
 excludePackages ['com.android', 'android.support','com.google'] 
```
after sync gradle this message show:
Error:(12, 0) Cannot cast object 'com.android' with class 'java.lang.String' to class 'int'

any help?

**Comment by seiginonakama** [comment]
could you provide your build.gradle ? I will have a try for this issue.

**Comment by SwiftyWang** [comment]
```
apply plugin: 'com.android.application'
apply plugin: 'com.google.gms.google-services'

configurations.all {
    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
}

android {
    compileSdkVersion rootProject.ext.compileSdkVersion
    buildToolsVersion rootProject.ext.buildToolsVersion
    lintOptions {
        checkReleaseBuilds false
        abortOnError false
    }
    defaultConfig {
        applicationId "com.xxx"
        minSdkVersion rootProject.ext.minSdkVersion
        targetSdkVersion rootProject.ext.targetSdkVersion

        versionCode 401
        versionName "6.054"

        multiDexEnabled true
        testInstrumentationRunner "com.xxx.test.LauncherInstrumentationTestRunner"
    }

    testOptions {
        unitTests.returnDefaultValues = true
    }

    dexOptions {
        javaMaxHeapSize "4g"
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
            applicationVariants.all { variant ->
                renameAPK(variant, defaultConfig)
            }
        }
    }

    packagingOptions {
        exclude 'AndroidManifest.xml'
        exclude 'META-INF/DEPENDENCIES.txt'
        exclude 'META-INF/DEPENDENCIES'
        exclude 'META-INF/dependencies.txt'
        exclude 'META-INF/LICENSE.txt'
        exclude 'META-INF/LICENSE'
        exclude 'META-INF/license.txt'
        exclude 'META-INF/LGPL2.1'
        exclude 'META-INF/NOTICE.txt'
        exclude 'META-INF/NOTICE'
        exclude 'META-INF/notice.txt'
        exclude 'DEPENDENCIES.txt'
        exclude 'DEPENDENCIES'
        exclude 'dependencies.txt'
        exclude 'LICENSE.txt'
        exclude 'LICENSE'
        exclude 'license.txt'
        exclude 'LGPL2.1'
        exclude 'NOTICE.txt'
        exclude 'NOTICE'
        exclude 'notice.txt'
    }
    sourceSets {
        main.java.srcDirs = ['src/main/java', 'src/main/kotlin']
        test.java.srcDirs = ['src/test/java', 'src/test/kotlin']
    }
}

def renameAPK(variant, defaultConfig) {
       ......
}

repositories {
    mavenCentral()
    jcenter()
    maven { url 'http://repo.brightcove.com/releases' }
    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
}

configurations.all {
    resolutionStrategy.force 'com.google.code.findbugs:jsr305:3.0.1'
}

public class BuildTimeListener implements TaskExecutionListener, BuildListener {
    private Clock clock
    private times = []

    @Override
    void beforeExecute(Task task) {
        clock = new org.gradle.util.Clock()
    }

    @Override
    void afterExecute(Task task, TaskState taskState) {
        def ms = clock.timeInMs
        times.add([ms, task.path])

        //task.project.logger.warn "${task.path} spend ${ms}ms"
    }

    @Override
    void buildFinished(BuildResult result) {
        println "Task spend time:"
        for (time in times) {
            if (time[0] >= 50) {
                printf "%7sms  %s\n", time
            }
        }
    }
    @Override
    void buildStarted(Gradle gradle) {}
    @Override
    void projectsEvaluated(Gradle gradle) {}
    @Override
    void projectsLoaded(Gradle gradle) {}
    @Override
    void settingsEvaluated(Settings settings) {}
}

project.gradle.addListener(new BuildTimeListener())

// When Dagger prints an error, data binding compilation will halt and sometimes print more than 100 errors, which is the default amount for javac.
gradle.projectsEvaluated {
    tasks.withType(JavaCompile) {
        options.compilerArgs << "-Xmaxerrs" << "500"
    }
}

dependencies {
    androidTestCompile 'com.android.support.test:runner:0.5', {
        exclude group: 'com.android.support', module: 'support-annotations'
    }
    androidTestCompile 'com.android.support.test:rules:0.5', {
        exclude group: 'com.android.support', module: 'support-annotations'
    }
    // Optional -- Hamcrest library
    androidTestCompile 'org.hamcrest:hamcrest-library:1.3', {
        exclude group: 'com.android.support', module: 'support-annotations'
    }
    // Optional -- UI testing with Espresso
    androidTestCompile("com.android.support.test.espresso:espresso-core:${rootProject.ext.espressoVersion}", {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5'
    testCompile "junit:junit:${rootProject.ext.junitVersion}"
    debugCompile 'com.amitshekhar.android:debug-db:1.0.0'
    debugCompile 'com.squareup.leakcanary:leakcanary-android:1.5'
    annotationProcessor "com.google.dagger:dagger-compiler:${rootProject.ext.daggerVersion}"
    annotationProcessor "com.google.dagger:dagger-android-processor:${rootProject.ext.daggerVersion}"
    annotationProcessor "com.jakewharton:butterknife-compiler:${rootProject.ext.butterknifeVersion}"
    releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5'
    compile fileTree(include: ['*.jar'], dir: 'libs')
    compile('com.crashlytics.sdk.android:crashlytics:2.5.5@aar') { transitive = true }
    compile 'com.android.support:multidex:1.0.1'
    compile 'com.google.zxing:core:3.2.0'
    compile 'org.pojava:datetime:3.0.2'
    compile 'me.grantland:autofittextview:0.2.1'
    compile 'com.github.castorflex.verticalviewpager:library:19.0.1'
    compile 'com.facebook.rebound:rebound:0.3.7'
    compile 'jp.wasabeef:recyclerview-animators:1.2.0@aar'
    compile 'com.pnikosis:materialish-progress:1.5'
    compile 'xyz.danoz:recyclerviewfastscroller:0.1.3'
    compile 'com.brightcove.player:android-sdk:4.+'
    compile project(':glmediaplayer')
    compile project(':launcher-commons')
}
```

**Comment by seiginonakama** [comment]
@SwiftyWang sorry, I found a wrong comment on doc, I missed '='......
```
excludePackages = ['com.android', 'android.support','com.google'] 
```

**Comment by seiginonakama** [comment]
@SwiftyWang  0.9.8.3 resolved this issue, you can have a try

**Comment by seiginonakama** [comment]
rootCause is the provide jars not join javaassit classpath

**Comment by seiginonakama** [comment]
resolved in 0.9.9
